presto-main/src/main/java/com/facebook/presto/server/security/SpnegoFilter.java

```diff
@@ -165,7 +165,8 @@ public Principal getUserPrincipal()
                         return;
                     }
                 }
-                catch (GSSException e) {
+                catch (Exception e) {
+                    LOG.debug(e, "Authentication failed for token %s", parts[1]);
```

Comment:
I don't think it's logged anywhere either. I spent some quality time with the debugger attached when I was trying stuff out writing the Kerberos docs.